src/ProductCounter/ProductCounterComponent.java

```diff
@@ -26,32 +27,12 @@
 	private ShoppingCart cart;
 	private ShoppingItem item;
 
-	public ProductCounterComponent() {
-		FXMLLoader fxmlLoader = new FXMLLoader(getClass().getResource("ProductCounterComponent.fxml"));
```

Comment:
Stötte även på problem här när `ProductCounterComponent` inkluderades från en annan `fxml`.

`fxml`-filen laddas först då den är inkluderas genom ex. `ProductItemComponent.fxml` för att därefter instansiera en controller (d.v.s. klassen `ProductCounterComponent`) blir det konstigt om man i konstruktorn laddar in samma `fxml` en gång till. Det skapas en oändlighetsloop.

Reply:
Det här beror också på `<fx:include />` vs `<ProductCounterComponent />`.

Reply:
`<ProductCounterComponent />` ser snyggare ut.